docs/guide/configuration/homeassistant.md

```diff
@@ -19,15 +19,17 @@ homeassistant:
   # Optional: Home Assistant status topic (default: shown below)
   # Note: in addition to the `status_topic`, 'homeassistant/status' will also be used
   status_topic: 'hass/status'
+  # Optional: Home Assistant legacy triggers (default: shown below), when enabled:
+  # - Zigbee2mqt will send an empty 'action' or 'click' after one has been send
+  # - A 'sensor_action' and 'sensor_click' will be discoverd
+  legacy_triggers: true
```

Comment:
Default is `true` indeed, but I propose to list `false` here (since we don't want users to use this anymore) and change ` (default: shown below)` to ` (default: true)`

Reply:
Oh upon re-reading your message, I understand you only propose to change the text.